lib/note-detail.jsx

```diff
@@ -19,36 +19,61 @@ export default React.createClass( {
 
 	componentWillMount: function() {
 		this.queueNoteSave = debounce( this.saveNote, saveDelay );
-		this.noteEditor = uninitializedNoteEditor;
+		this.focus = () => this.refs.editor.focus();
```

Comment:
It would be great to include some more context in the name here. Maybe something like `focusEditor`